backend/src/services/service-token/service-token-service.ts

```diff
@@ -142,7 +146,10 @@ export const serviceTokenServiceFactory = ({
     const updatedToken = await serviceTokenDAL.updateById(serviceToken.id, {
       lastUsed: new Date()
     });
-    return { ...serviceToken, lastUsed: updatedToken.lastUsed };
+
+    const organization = await orgDAL.findOrgByProjectId(serviceToken.projectId);
+
```

Comment:
Yes this is true. However I feel like we should try to standardize this as much as possible. If we don't make things consistent on the backend we'll end up having a lot of "know-how". 

Imagine someone creating a new endpoint, and they enable it for machine identities and service tokens. This endpoint depends on the `orgId` being provided from the auth middleware. When they go to test it with MI's it works as you would expect, but service tokens won't work for this endpoint now. 

This is still partially an issue, since we don't _(always)_ have an orgId on JWT, and API key auth. But moving forward I would say it makes sense to keep the data in the auth middleware as consistent as possible.